### PR TITLE
fix: force footer updates when pathname changes

### PR DESCRIPTION
**Pull request by taylrj**
This patch fixes a bug where the footer doesn't re-render when pathname changes,
so that the donation link contained in footer still remains stale query parameters (e.g. utm_campaign).

### CODE REVIEW

File: src/containers/app-shell.js [hunk 1]
```diff
@@ -47,14 +47,14 @@ const PinkBackgroundHeader = styled.div`
...
-const renderFooter = (footerType) => {
+const renderFooter = (footerType, pathname = '') => {
   switch(footerType) {
     case uiConst.footer.none: {
       return null
     }
     case uiConst.footer.default:
     default: {
-      return <Footer />
+      return <Footer key={pathname}/>
```
[the remark this answers is not in the document]
Comment (taylrj) [reply src/containers/app-shell.js]: Updated. 

Please see https://github.com/twreporter/twreporter-npm-packages/pull/135 as well.

**Comment by nickhsine** [comment]
@YuCJ 
Please help to review this PR and corresponding PR as well. Thanks.

**Comment by nickhsine** [comment]
@taylrj Please help to solve the PR conflicts.

**Review by YuCJ — COMMENTED** [review]
LGTM 👍

**Pull request merged**